### ISSUE TITLE
Missing Cake\Chronos\Date class

### ISSUE THREAD

**Issue by robertpustulka** [cakephp/chronos#408]
Shouldn't this class have an alias to a replacement?

https://github.com/cakephp/chronos/blob/3cbb9e2d8af1065c7189e1dc3f4fa641e797719f/src/Date.php#L15

FrozenDate is broken beacuse of this too:

https://github.com/cakephp/cakephp/blob/b58978b3433c3e1a8954a5d667896a8da2d93365/src/I18n/FrozenDate.php#L19

**Comment by LordSimal** [comment]
There is an alias present in the [ChronosDate class](https://github.com/cakephp/chronos/blob/2.x/src/ChronosDate.php#L288)

What exact error are you getting and how can we reproduce it?

**Comment by robertpustulka** [comment]
PHP Stan reported this issue in my project:

```
Call to static method now() on an unknown class Cake\Chronos\Date.   
         💡 Learn more at https://phpstan.org/user-guide/discovering-symbols

**Comment by LordSimal** [comment]
This is a stan warning, not a runtime error which one could interpret from your initial `FrozenDate is broken` message.

This was already discussed in https://github.com/cakephp/cakephp/issues/17229 and can't be fixed in this current patch release cycle.
But it will be fixed in the upcoming CakePHP 4.5 release because Frozendate will use the correct base class as you can see [here](https://github.com/cakephp/cakephp/blob/4.next/src/I18n/FrozenDate.php#L19-L29)

So unfortunately you will have to be patient and update to CakePHP 4.5 when it releases to fix this stan issue.
